README.md

```diff
@@ -930,6 +930,32 @@ some data, and it will attempt to resolve your provided path. If the path is
 valid, it will return the value residing there (`null` included!) in a `Just`.
 But if at any point that path "breaks" it will give you back a `Nothing`.
 
+#### propOr
+`crocks/helpers/propOr`
+```haskell
+propOr :: a -> (String | Number) -> b -> c
+```
+If you want some safety around pulling a value out of an Object or Array with a
+single key or index, you can always reach for `propOr`. Well, as long as you are
+working with non-nested data that is. Just tell `propOr` either the key or index
+you are interested in, and you will get back a function that will take anything
+and return the wrapped value if the key/index exists. If the
+key/index does not exist however, you will get back a default value.
```

Comment:
Not needed, but if you want you can add a brief example on these.
Only if you want to, this is not a requirement.

Reply:
I REALLY :heart: how you match the overall voice of the documents!
Good show m8!!

Reply:
Oh you just copied the original copy for those Maybe methods.
But either way awesome!!
:smile_cat:

Reply:
Yeah, was to say that. I am not so good on documentation 😅. But I can try change something if you think it is not a good one for `propOr` and `propPathOr`